test/tools/ossfuzz/abiV2FuzzerCommon.h

```diff
@@ -0,0 +1,56 @@
+#pragma once
+
+#include <libsolidity/interface/CompilerStack.h>
+#include <libyul/AssemblyStack.h>
+#include <liblangutil/Exceptions.h>
+#include <liblangutil/SourceReferenceFormatter.h>
+#include <libdevcore/Keccak256.h>
+
+namespace dev
+{
+namespace test
+{
+namespace abiv2fuzzer
+{
+class SolidityExecutionFramework
+{
+public:
+	SolidityExecutionFramework();
+	SolidityExecutionFramework(langutil::EVMVersion _evmVersion);
```

Comment:
```suggestion
	explicit SolidityExecutionFramework(langutil::EVMVersion _evmVersion = {});
```
(and remove the other constructor)

Reply:
done